autogen/agentchat/contrib/vectordb/base.py

```diff
@@ -171,7 +171,7 @@ def get_docs_by_ids(
             ids: List[ItemID] | A list of document ids. If None, will return all the documents. Default is None.
             collection_name: str | The name of the collection. Default is None.
             include: List[str] | The fields to include. Default is None.
-                If None, will include ["metadatas", "documents"], ids will always be included.
+                If None, will include ["metadata", "content"], ids will always be included.  # TODO - Confirm keys
```

Comment:
Changing this will lead to changes to all the other dbs. It's better to update the keys in the results of mongodb wrapper.